### PR TITLE
DTGB-426: Added table of contents to the preview template.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/51-templates/styleguide/styleguide.twig [hunk 1]
```diff
@@ -1,39 +1,82 @@
 <div class="{{ modifier }}">
     <h1>Styleguide</h1>
 
+    <h2>Table of contents</h2>
```
Comment (delrueba) [inline styleguide/components/51-templates/styleguide/styleguide.twig]: Should be in a nav region.
Identify which nav region it is by using aria-labelledby (refer to the heading element for instance)

see: https://inclusive-components.design/menus-menu-buttons/#tablesofcontent

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/51-templates/styleguide/styleguide.twig (hunk 1 above)
Comment (meirege) [reply styleguide/components/51-templates/styleguide/styleguide.twig]: Fixed it.

File: styleguide/components/51-templates/styleguide/styleguide.twig [hunk 2]
```diff
@@ -1,7 +1,8 @@
 <div class="{{ modifier }}">
     <h1>Styleguide</h1>
 
-    <h2>Table of contents</h2>
+    <nav aria-labelledby="table-of-contents-section">
```
Comment (delrueba) [inline styleguide/components/51-templates/styleguide/styleguide.twig]: Closing tag missing

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/51-templates/styleguide/styleguide.twig (hunk 2 above)
Comment (meirege) [reply styleguide/components/51-templates/styleguide/styleguide.twig]: Fixed.

**Review by delrueba — APPROVED** [review]

**Pull request merged**